src/Admin/AbstractAdmin.php

```diff
@@ -354,7 +354,7 @@ final public function getExportFields(): array
         return $fields;
     }
 
-    public function getDataSourceIterator(): SourceIteratorInterface
+    final public function getDataSourceIterator(): SourceIteratorInterface
```

Comment:
Did not check the stable branch, but we need this information using the `@final`annotation.